src/Features/Core/Portable/ImplementInterface/AbstractImplementInterfaceService.CodeAction.cs

```diff
@@ -82,7 +84,15 @@ internal partial class ImplementInterfaceCodeAction : CodeAction
                 State state,
                 ISymbol throughMember)
             {
-                return new ImplementInterfaceCodeAction(service, document, state, explicitly: false, abstractly: false, throughMember: throughMember);
+                return new ImplementInterfaceCodeAction(service, document, state, explicitly: false, abstractly: false, onlyUnimplemented: false, throughMember: throughMember);
+            }
+
+            public static ImplementInterfaceCodeAction CreateImplementRemainingExplicitlyCodeAction(
+                AbstractImplementInterfaceService service,
+                Document document,
+                State state)
+            {
+                return new ImplementInterfaceCodeAction(service, document, state, explicitly: true, abstractly: false, onlyUnimplemented: true, throughMember: null);
```

Comment:
the naming would fit better here as well. the caller is `ImplementRemainingExplicitly`.  it would make sense for that to split to two bool values.